olive/workflows/run/run.py

```diff
@@ -124,6 +124,10 @@ def run(config: Union[str, Path, dict], setup: bool = False):
     # input model
     input_model = config.input_model.create_model()
 
+    # Azure ML Client
+    if config.azureml_client:
+        config.engine.azureml_client = config.azureml_client
```

Comment:
shouldn't the config happens in config json?

Reply:
could not the following works?

"azureml_client": {
    "subscription_id": "<place_holder>",
    "resource_group": "<place_holder>",
    "workspace_name": "<place_holder>"
},
"engine": {
        "search_strategy": {
            "execution_order": "joint",
            "search_algorithm": "random",
            "search_algorithm_config": {
                "num_samples": 3,
                "seed": 0
            }
        },
        "evaluator": "common_evaluator",
        "host": "local_system",
        "target": "local_system",
        "azureml_client": "azureml_client"
        "clean_cache": true,
        "cache_dir": "cache"
    }

Reply:
Please consider that case that there are multiple workspaces in one olive run in future since we all not sure the assumption that only use one workspace is right.

Reply:
I read that aml documentation. So the question here is, will Olive handle the sharing operation? Like if the user wants to use the model registered in another workspace, should they complete the sharing operation already?

Reply:
not just about share model. Please consider the case: model is registered in workspace A. but compute target is in workspace B. 

Since no one know whether our current assumption is correct or not, we'd better to keep that in mind when we design/implement to consider the extensibility.

Reply:
I just realize that the assets sharing is between workspace and registry but not workspace and workspace. So in this case there will only be one workspace in total.

Reply:
We can add support registry support into AMLClient in the future.